scss/variables.scss

```
@@ -0,0 +1,80 @@
+//font-family//
```

scss commentaries is looks like
```suggestion
/* font-family */
```